kafka_versions.yaml

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can we add a comment explaining what this file is, and importantly, what consumes its content?

Reply:
updated :)